### PR TITLE
rename fields of ProcessedEnvelope

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

rename fields of ProcessedEnvelope.
orchestrator changes will go after this

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by mdayican** [comment]
> it should be one of those IDs. strict XOR. because we are migrating from `id` to `envelope_id`. same goes to log messages.
> 
> don't purely count on me, verify with another reviewer

I am keeping all Ids now, than orchestrator start sending envelope_id  and come back here and delete id

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/in/msg/ProcessedEnvelope.java [hunk 1]
```diff
@@ -9,16 +9,22 @@
...
+        @JsonProperty("envelope_id") UUID envelopeId,
+        @JsonProperty("ccd_id") Long ccdId,
+        @JsonProperty("envelope_ccd_action") String envelopeCcdAction
     ) {
         this.id = id;
-        this.processedCcdReference = processedCcdReference;
-        this.processedCcdType = processedCcdType;
+        this.envelopeId = envelopeId;
+        this.ccdId = ccdId;
+        this.envelopeCcdAction = envelopeCcdAction;
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/in/msg/ProcessedEnvelope.java:28]: Shouldn't `processedCcdReference` map to the same field as `ccdId` and `processedCcdType` to the same field as `envelopeCcdAction`? This way we could still reference the same fields, the change would be minimal. Once the orchestrator starts sending the new fields, we'd just need to remove the old ones.

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/in/msg/ProcessedEnvelope.java:28]: processor does not do anything with new fields.

**Comment by lgonczar** [comment]
> it should be one of those IDs. strict XOR. because we are migrating from `id` to `envelope_id`. same goes to log messages.
> 
> don't purely count on me, verify with another reviewer

As just discussed with Mustafa, orchestrator will always include `id` in the message, as long as the processor uses it. The two other fields aren't used yet, anyway.

**Pull request merged**